backend/database/versions/b65dad1f2a7e_add_explorer_field_to_dataset.py

```diff
@@ -20,7 +20,10 @@ def upgrade():
     # ### commands auto generated by Alembic - please adjust! ###
     op.add_column("dataset", sa.Column("explorer_url", sa.String(), nullable=True))
     op.create_index(op.f("ix_dataset_explorer_url"), "dataset", ["explorer_url"], unique=False)
-    op.execute("UPDATE dataset SET explorer_url=deployment_directory.url FROM deployment_directory WHERE deployment_directory.dataset_id=dataset.id;")  # noqa E501
+    op.execute(
```

Comment:
can you remove this change?

Reply:
I got lint error for the line being too long 😭

Reply:
Actually reverting the changes in this file worked 😆 Probably because it already has `# noqa E501` to ignore row length? But now I'm unsure what made PR lint freak out about this file in the first place -.- 

Thanks for helping me troubleshoot, Madison!!